ArduCopter/config.h

```diff
@@ -544,12 +544,16 @@
 #endif
 
 // default maximum vertical velocity and acceleration the pilot may request
-#ifndef PILOT_VELZ_MAX
+#ifndef PILOT_VELZ_MAX              // Depreciated can be removed.
  # define PILOT_VELZ_MAX    250     // maximum vertical velocity in cm/s
 #endif
 #ifndef PILOT_ACCEL_Z_DEFAULT
  # define PILOT_ACCEL_Z_DEFAULT 250 // vertical acceleration in cm/s/s while altitude is under pilot control
 #endif
+#ifndef PILOT_SPEED_DEF
+ # define PILOT_SPEED_DEF    250     // maximum vertical velocity in cm/s
+#endif
```

Comment:
I think we don't need to change the macro name, but if you want to do that, change the old macro name, don't add a new one and leave the old, unused, one.

Reply:
I've reverted to using the old name, removed the PILOT_SPEED_DEF and removed the comment deprecating the original name.